README.md

```diff
@@ -1,32 +1,52 @@
 # ChromeIDE
 Parallax microcontroller development environment based on Chrome application technologies.
 
-## Installing from Source
+## Build from Source
 
-1. Clone this respository.  Choose your file destination from Terminal and
+This is a one-time procedure needed to initially run the application.  To build the application will need the Node javascript library.  If you don't have it, then:
 
-	``` git clone git@github.com:parallaxinc/ChromeIDE ```
+1. Install Node by going to [nodejs.org/download](https://nodejs.org/download/) and selecting the option that is right for your system.
```

Comment:
We are going to need to mention node-gyp and it's system-specific dependencies.  See https://github.com/TooTallNate/node-gyp#installation - note that it comes bundled with node/npm so node-gyp doesn't need to be installed but the dependencies do.